### PR TITLE
ExternalLink widget

### LINKED ISSUE #1671: Extensibility: Details View: Hyperlinks to external websites
**Description**
I would like to have a way to link to external websites in the details view as a form of widget or maybe just plain text that is a valid URL.

**Reasons**
The TrivyReport resource holds a list of CVEs that were found when scanning pods in the cluster. Each CVE entry has information such as name, version, fix version, package and a list of references which are links to external sites where the CVE was defined in detail. It would be much easier for people using this view if the URL were clickable so that they do not need to Ctrl+C/Ctrl+V the URL in their browser so they can see more information. 

**Attachments**
image

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**
Closes #1671

### CODE REVIEW

**Comment by wchrapka** [comment]
Please update the EXT docs with the new widget

File: core-ui/src/components/Extensibility/components/Externallink.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      return expression.evaluate();
+    } catch (e) {
+      console.warn('Widget::shouldBeVisible error:', e);
+      return { error: e };
+    }
+  }
+
+  let href;
+  if (typeof value === 'string') {
+    href = value.substring(0, 8) === 'https://' ? value : `https://${value}`;
```
Comment (wchrapka) [inline core-ui/src/components/Extensibility/components/Externallink.js:28]: it works well, but there's a special fn in js for that [value.startsWith('https://')](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith)

File: docs/extensibility/display-section.md [hunk 2]
```diff
@@ -311,6 +312,43 @@ ControlledBy widgets render the kind and the name with a link to the resources t
 
 <img src="./assets/display-widgets/ControlledBy--kindOnly.png" alt="Example of a ControlledBy widget without name link" width="40%" style="border: 1px solid #D2D5D9">
 
+### ExternalLink
+
+ExternalLink widgets render the link to external page.
+
+#### Widget-specific parameters
+
+- **linkFormula** - an optional jsonata function to generate custom link.
```
Comment (grego952) [inline docs/extensibility/display-section.md:321]:
```suggestion
- **linkFormula** - an optional JSONata function to generate a custom link.
```
Do we have a link to this JSON function? Maybe we can add it here?

File: docs/extensibility/display-section.md [hunk 3]
```diff
@@ -311,6 +312,43 @@ ControlledBy widgets render the kind and the name with a link to the resources t
...
+- **textFormula** - an optional string or item which will be displayed as link.
```
Comment (grego952) [inline docs/extensibility/display-section.md:322]:
```suggestion
- **textFormula** - an optional string or item which is displayed as a link.
```

File: docs/extensibility/display-section.md [hunk 4]
```diff
@@ -311,6 +312,43 @@ ControlledBy widgets render the kind and the name with a link to the resources t
...
+
+#### Examples
+
+##### linkFormula and textFormula usage
+
+```json
```
Comment (grego952) [inline docs/extensibility/display-section.md:328]: Can you change the code examples to yaml?
See https://github.com/kyma-project/busola/issues/1873 for reference

File: docs/extensibility/display-section.md [hunk 5]
```diff
@@ -311,6 +312,43 @@ ControlledBy widgets render the kind and the name with a link to the resources t
...
+  "linkFormula": "'https://' & $item.port.name & ':' & $string($item.port.number)",
+  "textFormula": "$item.port.name"
+}
+```
+
+<img src="./assets/display-widgets/ExternalLink.png" alt="Example of a ExternalLink widget" width="50%" style="border: 1px solid #D2D5D9">
+
+##### Source only
+
+```json
```
Comment (grego952) [inline docs/extensibility/display-section.md:342]: See my previous comment

**Review by wchrapka — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**